Dockerfile

```diff
@@ -5,6 +5,9 @@ ENV UWSGI_INI /srv/www/yogsite/uwsgi.ini
 COPY . /srv/www/yogsite
 COPY nginx.conf /etc/nginx/sites-available/ 
 
+RUN apt-get install -y ffmpeg
```

Comment:
Seems like current available version on Debian is ffmpeg v4.1.6 which is currently vulnerable to a RCE vulnerability, see https://www.cvedetails.com/cve/CVE-2021-30123/

Reply:
How do I improve this situation

Reply:
There isnt really an ideal solution until latest version of ffmpeg is moved into the stable branch.

You can use the experimental branch, but who knows what issues that might have.

```
RUN echo "deb http://deb.debian.org/debian experimental main" >>  /etc/apt/sources.list
RUN apt-get update
RUN apt-get install -y -t experimental ffmpeg
```

Or see if theres some alternative / workaround... I couldn't see any listing for a workaround on the CVE though.

Reply:
Seems like there are solutions to converting to ogg format which can run in browser.

https://blog.addpipe.com/using-webaudiorecorder-js-to-record-audio-on-your-website/

Reply:
I still need to use ffmpeg to verify the duration of the sound clip to make sure the user didn't upload like a long-ass sound clip or something

Reply:
Alright after doing some testing I was able to get the experimental version of ffmpeg to install properly